scanner/alma.go

```diff
@@ -0,0 +1,118 @@
+package scanner
+
+import (
+	"github.com/future-architect/vuls/config"
+	"github.com/future-architect/vuls/logging"
+	"github.com/future-architect/vuls/models"
+)
+
+// inherit OsTypeInterface
+type alma struct {
+	redhatBase
+}
+
+// NewAmazon is constructor
```

Comment:
Isn't this a typo? (NewAmazon → NewAlma)

Reply:
I fixed